examples/TF_1_x/README.md

```diff
@@ -0,0 +1,114 @@
+# Intel<sup>®</sup> OpenVINO<sup>TM</sup> integration with TensorFlow Python Object Detection Example
+
+These examples demonstrate how to use **Intel<sup>®</sup> </sup> OpenVINO<sup>TM</sup> integration with Tensorflow** to recognize and detect objects in images and videos.
+
+## Demos showcased in the examples
+
+ Object detection demo uses YOLOv3 daraknet model to detect objects in a given image, video, directory and camera input.
+
+## Setup for the examples
+
+Before you proceed to run the examples, you will have to clone the `openvino_tensorflow` repository to your local machine. For this, run the following commands:
+
+```bash
+$ git clone https://github.com/openvinotoolkit/openvino_tensorflow.git
+$ cd openvino_tensorflow
+$ git submodule init
+$ git submodule update --recursive
+```
+
+## Python implementation for object detection
+
+For this example, we assume that you've already:
+
+* Installed TensorFlow on your system
+* Installed **Intel<sup>®</sup> </sup> OpenVINO<sup>TM</sup> integration with Tensorflow** on your system
+
+Refer to [**this page**](https://github.com/openvinotoolkit/openvino_tensorflow#installation) for a quick install with pip.
+
+Install the pre-requisites
+```bash
+$ cd <path-to-openvino_tensorflow-repository>/examples/TF_1_x
+$ pip3 install -r requirements.txt
+```
+
+The TensorFlow Yolo v3 model used in this demo is not packaged in the repository because of its size. So, follow the instructions below to convert the model from DarkNet to TensorFlow and download the labels and weights to the `data` directory in your `cloned repo of openvino_tensorflow`:
+
+Please note: The instructions below should not be executed in an active virtual environment. The convert_yolov3.sh script activates a python virtual environment for conversion.
+
+```bash
+$ cd <path-to-openvino_tensorflow-repository>/examples
+$ chmod +x convert_yolov3.sh
+$ ./convert_yolov3.sh
+```
+
+Once completed, the data folder will contain following files needed to run the object detection example:
+
+* coco.names
+* yolo_v3_darknet.pb
+
+Run the object detection example using the instructions below:
+
+```bash
+$ cd <path-to-openvino_tensorflow-repository>
+$ python3 examples/TF_1_x/object_detection_sample.py
+```
+
+This uses the default example image that comes with this repository, and should
+output something similar as below:
+
+<p align="left">
+  <img src="../examples/TF_1_x/data/detections.jpg" width="200" height="200"
```

Comment:
we dont need new data folder in TF_1_x. Existing data folder in examples should be leveraged.